### PR TITLE
Add visualization of directed edges in a directed graph

### PR DESCRIPTION
**Pull request by spisladqo**
#### Create vector representation of edges.

Add `isDirected` flags to main screen, graph and edge view models, this chain allows to display arrows if a graph is directed and don't display them if graph is undirected. 

Calculations for finding arrow position happen only in a directed graph.

Move all logic from edge view to edge view model.

Make small refactor.

### CODE REVIEW

**Review by qrutyy — CHANGES_REQUESTED** [review]
Good job, but please checkout from the changes, that i've made earlier) They will be released in main, after your approval) Thx!

**Review by shakareem — APPROVED** [review]
niceniceniceniceniceEXCELLENT

File: app/src/main/kotlin/viewmodel/graph/GraphViewModel.kt [hunk 1]
```diff
@@ -64,9 +65,8 @@ class GraphViewModel<D>(
         val edge = graph.addEdge(firstVertexVM.vertex, secondVertexVM.vertex)
 
         _edgeViewModels = _edgeViewModels.toMutableMap().apply {
-            this[edge] = EdgeViewModel(firstVertexVM, secondVertexVM, edge)
+            this[edge] = EdgeViewModel(firstVertexVM, secondVertexVM, isDirected)
         }
-        TestRepresentation().place(740.0, 650.0, verticesVM)
```
Comment (qrutyy) [inline app/src/main/kotlin/viewmodel/graph/GraphViewModel.kt:69]: Is this necessary? I mean, the deletion

Comment (qrutyy) [reply app/src/main/kotlin/viewmodel/graph/GraphViewModel.kt:69]: I think its quite logic, to update representation after add

Comment (spisladqo) [reply app/src/main/kotlin/viewmodel/graph/GraphViewModel.kt:69]: I don't remember exactly what was the reason I deleted it, but it works without it just fine

**Pull request merged**